samples/cpp/benchmark_app/inputs_filling.cpp

```diff
@@ -157,17 +158,18 @@ ov::Tensor create_tensor_from_binary(const std::vector<std::string>& files,
         std::ifstream binaryFile(files[inputIndex], std::ios_base::binary | std::ios_base::ate);
         OPENVINO_ASSERT(binaryFile, "Cannot open ", files[inputIndex]);
 
-        auto fileSize = static_cast<std::size_t>(binaryFile.tellg());
-        binaryFile.seekg(0, std::ios_base::beg);
-        OPENVINO_ASSERT(binaryFile.good(), "Can not read ", files[inputIndex]);
         auto inputSize = tensor_size * sizeof(T) / binaryBatchSize;
-        OPENVINO_ASSERT(fileSize == inputSize,
-                        "File ",
-                        files[inputIndex],
-                        " contains ",
-                        fileSize,
-                        " bytes, but the model expects ",
-                        inputSize);
+
+        std::string extension = get_extension(files[inputIndex]);
+        if (extension == "npy") {
+            slog::info << "Prepare numpy file " << files[inputIndex] << slog::endl;
+            verifyNumpyFile<T>(binaryFile, files[inputIndex], inputInfo.dataShape, inputSize);
+        } else if (extension == "bin") {
+            slog::info << "Prepare binary file " << files[inputIndex] << slog::endl;
```

Comment:
Is it needed to print this for bin and npy? benchmark logs are already overwhelming. You when you report an error in `verifyNumpyFile()` or `verifyBinaryFile()` you report file name anyway.
I think the prints should be removed

Reply:
You are right, removed!